Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/VmServiceWrapper.java

```diff
@@ -369,16 +366,13 @@ public void received(final Stack vmStack) {
         ApplicationManager.getApplication().executeOnPooledThread(() -> {
           InstanceRef exceptionToAddToFrame = exception;
 
-          ElementList<Frame> elementList;
-          if (RENDER_CAUSAL_FRAMES) {
+          ElementList<Frame> elementList = vmStack.getAwaiterFrames();
```

Comment:
For this particular change, older versions of the protocol won't send in await frames (or async causal frames). Those fields will just not be there when we go to get them out of the json. So we can degrade gracefully here w/o version checks.

(similarly, even VMs which do support sending the new frames types don't send at paused locations where they're not necessary - purely sync code)